include/wx/localedefs.h

```diff
@@ -108,6 +108,83 @@ struct WXDLLIMPEXP_BASE wxLanguageInfo
     const char* TrySetLocale() const;
 };
 
+// ----------------------------------------------------------------------------
+// wxLocaleIdent: allows to fully identify a locale under all platforms
+// ----------------------------------------------------------------------------
+
+class WXDLLIMPEXP_BASE wxLocaleIdent
+{
+public:
+    // Leave language empty
+    wxLocaleIdent() { }
+
+    // Construct from language, i.e. a two-letter ISO 639-1 code (or a
+    // three-lette ISO 639-2 code if there is no ISO 639-1 code for this
```

Comment:
```suggestion
    // three-letter ISO 639-2 code if there is no ISO 639-1 code for this
```

Reply:
Thanks for noticing this! I wonder why this wasn't caught by our spelling check...